clickControl/clickControl.py

```diff
@@ -132,272 +190,288 @@ def updateLinks(self, msg, links=[], delays=[], capacities=[], losses=[]):
+            self.ccp.parse(self._conf_path)
+            args = {'timeout': timeout, 'packets': packets, 'sampling_prob': sampling_prob}
+            self.log.info('setting simple reorder config: %s', args)
             for key, value in args.iteritems():
                 if value is not None:
                     if not self.ccp.set_value(node, key, value):
-                        self.log.info('Error setting {} --> {} in updateSimpleReorder'.format(key, value))
+                        self.log.info('Error setting %s --> %s in updateSimpleReorder', key, value)
                         return False
 
-            if not self.ccp.set_value(node, 'active', str(active).lower()):  # lower case bool in click
-                self.log.info('Error setting simple reorder link active to {}'.format(active))
+            if not self.ccp.set_value(node, 'active', str(active).lower()):  # lower case in click
+                self.log.info('Error setting simple reorder link active to %s', active)
                 return False
 
-        except ClickConfigParserException as e:
-            self.log.error(e)
+        except ClickConfigParserException as err:
+            self.log.error(err)
             return False
 
         return True
 
     @agentmethod()
-    def updateRoute(self, msg, router="", ip="", port="", next_hop=""):
-        m = re.match("[0-9]+", router)
-        if m:
-            router = "router%s" % m.group(0)
+    # pylint: disable=too-many-arguments
+    def updateRoute(self, msg, router="", ip_addr="", port="", next_hop=""):
+        re_match = re.match("[0-9]+", router)
+        if re_match:
+            router = "router%s" % re_match.group(0)
 
         if port == "":
-            m = re.match("[0-9]+", next_hop)
-            if m:
-                next_hop = "router%s" % m.group(0)
-            port = self.cg.getPort(router, next_hop)
+            re_match = re.match("[0-9]+", next_hop)
+            if re_match:
+                next_hop = "router%s" % re_match.group(0)
+            port = self.clg.getPort(router, next_hop)
             if not port:
-                self.log.error("Click: Cannot find link between %s and %s\n" % (router, next_hop))
+                self.log.error("Click: Cannot find link between %s and %s\n", router, next_hop)
                 return False
-            
-        return self.updateClickConfig(msg, router, 'set', '{} {}'.format(ip, port))
+
+        return self.updateClickConfig(msg, router, 'set', '{} {}'.format(ip_addr, port))
 
     @agentmethod()
-    def updateRoutes(self, msg, path=[], ip=""):
-        c = 0
-        for c in range(len(path) - 1):
-            router = path[c]
-            next_hop = path[c + 1]
-            self.updateRoute(msg, router=router, ip=ip, next_hop=next_hop)
-            
+    # pylint: disable=dangerous-default-value
+    def updateRoutes(self, msg, path=[], ip_addr=""):
+        for path_number in range(len(path) - 1):
```

Comment:
Do the for X in enumrate(..) thing here as well for consistency?

Reply:
good call.